website/src/pages/docs/getting-started/esm-typescript-usage.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: ESM TypeScript Usage
+---
+
+# ESM TypeScript Usage
+
+ESM is an important step in the JavaScript ecosystem because it allows static analysis of the depency tree.
+Forr bundlers such as Webpack this is a huge as it allows the complied bundle to only contain code that is required at runtime, which means smaller download size when loading code over the network.
```

Comment:
```suggestion
For bundlers such as Webpack, this is huge as it allows the compiled bundle only to contain code that is required at runtime, which means a smaller download size when loading code over the network.
```

Reply:
resolve as i accidentally first committed @saihaj comment, that missed a typo and had to apply it manually